pkg/fftypes/transaction.go

```diff
@@ -19,8 +19,10 @@ package fftypes
 type TransactionType = FFEnum
 
 var (
-	// TransactionTypeNone indicates no transaction should be used for this message/batch
+	// TransactionTypeNone deprecreated - replaced by TransactionTypeUnpinned
```

Comment:
typo: deprecated

Reply:
Looking forward to the day we `dep-recreate` something